packages/forms-web-app/src/validators/register/behalf/organisation-name.js

```diff
@@ -0,0 +1,9 @@
+const { body } = require('express-validator');
+
+const rules = () => {
+  return [body('organisation-name').notEmpty().withMessage('Enter your organisation name')];
```

Comment:
Sorry, another change. Error message should be **Enter the name of the organisation you work for**

Reply:
Changed.